libraries/botbuilder-dialogs-adaptive/etc/botbuilder-dialogs-adaptive.api.md

```diff
@@ -1200,6 +1200,7 @@ export class HttpRequest<O extends object = {}> extends Dialog<O> implements Htt
     body?: ValueExpression;
     contentType?: StringExpression;
     disabled?: BoolExpression;
+    EndDialogWithResultAsync(dc: DialogContext, result: Result, traceInfo: any, label: string): Promise<DialogTurnResult<any>>;

```

Comment:
This method should be private https://github.com/microsoft/botbuilder-dotnet/pull/6404/files#diff-3497e92e8b8656df453ce18e36066ebbb50f5aaac85f94e28eb6d106a536b165R424